ursgal/wrappers/thermo_raw_file_parser_1_1_2.py

```diff
@@ -123,3 +124,9 @@ def preflight(self):
                 self.params["command_list"].append("{0}".format(flag))
 
         return self.params
+
+
+    def postflight(self):
+        if self.params['prefix'] != '':
+            shutil.move(self.params['translations']['output_file_incl_path'].replace(self.params['prefix'] +'_', ''), self.params['translations']['output_file_incl_path'] )
```

Comment:
"replace" might not be the best option here, since it would replace the prefix string everywhere in the path
--> make sure to only replace the first instance in the basename